src/Core/Manifest/ClassManifest.php

```diff
@@ -382,15 +384,6 @@ public function getOwnerModule($class)
     }
 
     /**
-     * Used to set up files that we want to exclude from parsing for performance reasons.
```

Comment:
@dhensby wrote an improved class parser recently that discarded class-body. This is probably the reason. :D